dictionary.js

```diff
@@ -0,0 +1,44 @@
+const dictionary = [];
+
+function addWord(event) {
+    event.preventDefault();
+    const word = add.value;
+    if (!dictionary.includes(word)) {
+        dictionary.push(word);
+        const listElem = list;
+        const newWord = document.createElement('li');
+        newWord.textContent = word;
+        listElem.appendChild(newWord);
+        newWord.style.display = 'none';
+        alert("You have successfully added '" + word + "' to the dictionary.");
```

Comment:
It is not recommended to talk to the user through errors. Try to display a message instead.